pkg/controllers/subnetport/addressbinding_webhook.go

```diff
@@ -0,0 +1,68 @@
+package subnetport
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"reflect"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/apis/vpc/v1alpha1"
+	"github.com/vmware-tanzu/nsx-operator/pkg/util"
+)
+
+// log is for logging in this package.
+var addressbindinglog = logf.Log.WithName("addressbinding-webhook")
```

Comment:
addressbindinglog doesn't log anything, refer https://github.com/vmware-tanzu/nsx-operator/blob/topic/zhengxie/v4.2.0_webhook_log/pkg/controllers/subnetset/subnetset_webhook.go#L74, directly use the global log  here and below.

Reply:
Fixed.